archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/BackportingConfigProxyFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.archaius.bridge;
+
+import com.netflix.archaius.ConfigProxyFactory;
+import com.netflix.archaius.api.Config;
+import com.netflix.config.ConfigurationManager;
+
+public class BackportingConfigProxyFactory extends ConfigProxyFactory {
+
+    public BackportingConfigProxyFactory(Config config) {
+        super(config);
+    }
+    
+    @Override
+    protected <T> MethodInvoker<T> createDynamicProperty(Class<T> type, String propName, String defaultValue) {
+        if(defaultValue != null && getConfig().getRawProperty(propName) == null)
```

Comment:
where's getConfig() from? I don't see it in the base class. Also, typo on the == null?